tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "lib": [ "dom", "es7" ],
+    "sourceMap": true,
```

Comment:
Does this generate sourcemaps for production? Does that add to the size of the deliverable?

Reply:
I believe this depends on the webpack config (and is configurable). I'll check and reply here